modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -53,3 +53,7 @@
 		margin-bottom: 0.5rem;
 	}
 }
+
+.dropdown-menu__languages {
+	max-height: 295px;
+}
```

Comment:
Just referencing [prior discussion about this pattern](https://github.com/liferay-frontend/liferay-portal/pull/999/files) from your other pull, in case anybody wants to look.